<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>403</width>
    <height>469</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(56, 55, 66);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QStackedWidget" name="stackedWidget">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>0</y>
      <width>421</width>
      <height>401</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color: rgb(255, 255, 255);</string>
    </property>
    <widget class="QWidget" name="start_Page">
     <widget class="QLabel" name="title">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>30</y>
        <width>251</width>
        <height>81</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <family>OCR A Extended</family>
        <pointsize>18</pointsize>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 255, 247);</string>
      </property>
      <property name="text">
       <string>EPIDEMIONATOR</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_default">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>130</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Default Data</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="pushButton_custom">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>180</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Custom Data</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="pushButton_quit">
      <property name="geometry">
       <rect>
        <x>160</x>
        <y>250</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Quit</string>
      </property>
      <attribute name="buttonGroup">
       <string notr="true">buttonGroup</string>
      </attribute>
     </widget>
     <widget class="QPushButton" name="pushButton_about">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>320</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>About</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="default_Page_01">
     <widget class="QPushButton" name="pushButton_back_01">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_17">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>321</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Geben Sie den Pfad der Textdatei an:</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_selectDir02">
      <property name="geometry">
       <rect>
        <x>130</x>
        <y>250</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Select Directory</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="custom_Page_01">
     <widget class="QPushButton" name="pushButton_back_02">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_selectDir01">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>210</y>
        <width>131</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Select Directory</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_10">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>80</y>
        <width>321</width>
        <height>111</height>
       </rect>
      </property>
      <property name="font">
       <font>
        <pointsize>8</pointsize>
       </font>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Hier wird das jeweilige Projekt angelegt und benannt</string>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="custom_Page_02">
     <widget class="QPushButton" name="pushButton_back_03">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_11">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>70</y>
        <width>251</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Bitte geben Sie Ihre Daten ein:</string>
      </property>
     </widget>
     <widget class="QTableWidget" name="tableWidget_Overview">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>190</y>
        <width>261</width>
        <height>211</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_continue_02">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>370</y>
        <width>91</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Continue</string>
      </property>
     </widget>
     <widget class="QWidget" name="horizontalLayoutWidget_2">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>100</y>
        <width>361</width>
        <height>86</height>
       </rect>
      </property>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLineEdit" name="lineEdit_Cityname">
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="text">
              <string>Stadtname</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <item>
            <widget class="QLineEdit" name="lineEdit_Population">
             <property name="styleSheet">
              <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);

</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_13">
             <property name="text">
              <string>Einwohnerzahl</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton_addCity">
         <property name="styleSheet">
          <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
         </property>
         <property name="text">
          <string>add City</string>
         </property>
         <property name="autoDefault">
          <bool>true</bool>
         </property>
         <property name="default">
          <bool>true</bool>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </widget>
    <widget class="QWidget" name="custom_Page_03">
     <widget class="QPushButton" name="pushButton_back_04">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>30</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Back</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_14">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>80</y>
        <width>151</width>
        <height>16</height>
       </rect>
      </property>
      <property name="text">
       <string>Insert Connections!</string>
      </property>
     </widget>
     <widget class="QLabel" name="label_cityToConnect">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>120</y>
        <width>141</width>
        <height>81</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_addCon">
      <property name="geometry">
       <rect>
        <x>140</x>
        <y>220</y>
        <width>101</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>add Connection</string>
      </property>
      <property name="autoDefault">
       <bool>true</bool>
      </property>
      <property name="default">
       <bool>true</bool>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_nextC">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>220</y>
        <width>101</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>next City</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_continue_03">
      <property name="geometry">
       <rect>
        <x>300</x>
        <y>360</y>
        <width>101</width>
        <height>28</height>
       </rect>
      </property>
      <property name="styleSheet">
       <string notr="true">color: rgb(0, 0, 0);
background-color: rgb(255, 255, 255);</string>
      </property>
      <property name="text">
       <string>Continue</string>
      </property>
     </widget>
     <widget class="QWidget" name="verticalLayoutWidget_3">
      <property name="geometry">
       <rect>
        <x>40</x>
        <y>110</y>
        <width>201</width>
        <height>101</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_7">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_4">
         <item>
          <widget class="QLabel" name="label_15">
           <property name="text">
            <string>City:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_city">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <item>
          <widget class="QLabel" name="label_16">
           <property name="text">
            <string>Distance: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_distance">
           <property name="styleSheet">
            <string notr="true">background-color: rgb(255, 255, 255);
color: rgb(0, 0, 0);</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>403</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
 <buttongroups>
  <buttongroup name="buttonGroup"/>
 </buttongroups>
</ui>
